File: Nível 1/2381 - Lista de Chamada/main.lua
alunos, vencedor = io.read():match("(%S+)%s+(%S+)")
nomes = {}

for i = 1, alunos do 
    nomesAluno = io.read()
    table.insert(nomes, nomesAluno)
end

table.sort(nomes)
print(nomes[tonumber(vencedor)])

File: Nível 1/2381 - Lista de Chamada/main.py
alunos, vencedor = map(int, input().split())
nomes = []

for i in range(1, alunos + 1):
    nomeAluno = str(input())
    nomes.append(nomeAluno)

nomes.sort()

alunoPremiado = vencedor - 1
print(nomes[alunoPremiado])